aepsych/benchmark/test_functions.py

```diff
@@ -110,26 +112,28 @@ def make_songetal_testfun(
     """
     valid_phenotypes = ["Metabolic", "Sensory", "Metabolic+Sensory", "Older-normal"]
     assert phenotype in valid_phenotypes, f"Phenotype must be one of {valid_phenotypes}"
-    x = dubno_data[dubno_data.phenotype == phenotype].freq.values
-    y = dubno_data[dubno_data.phenotype == phenotype].thresh.values
+    x = torch.tensor(dubno_data[dubno_data.phenotype == phenotype].freq.values, dtype=torch.float64)
+    y = torch.tensor(dubno_data[dubno_data.phenotype == phenotype].thresh.values, dtype=torch.float64)
+
     # first, make the threshold fun
     threshfun = make_songetal_threshfun(x, y)
 
     # now make it into a test function
-    def song_testfun(x, cdf=False):
+    def song_testfun(x: torch.Tensor, cdf: bool = False) -> torch.Tensor:
```

Comment:
These are private functions, I'm not sure if adding typehints to them does anything, though they are returned so I'm not sure. Nonetheless, it's odd that this one has typehints but the above one doesn't.

Reply:
Okay, I will remove their type hints, as it is unnecessary.